backend/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "backend"
+version = "0.1.0"
+default-run = "backend"
+edition = "2021"
+description = "Bob Management GUI: Backend"
+authors = ["Romanov Simeon ArchArcheoss@proton.me", "Artemev Ilia", "Riadinskii Kirill"]
+license = "Unlicense"
+repository = "https://github.com/qoollo/bob-management"
+publish = false
+keywords = [ "BOB", "Management", "GUI" ]
+readme = "./README.md"
+
+[dependencies]
+# Axum related
+axum = "0.6"
+axum-macros = "0.3"
+axum-login = "0.5"
+axum-sessions = "0.5"
+tower = "0.4"
+tower-http = { version = "0.4", features = ["cors"] }
+
+# Logging
+tracing = "0.1"
+tracing-subscriber = "0.3"
+
+# Error Handling
+color-eyre = "0.6"
+thiserror = "1.0"
+
+# General
+tokio = { version = "1.0", features = ["full"] }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+serde_yaml = "0.9"
+serde_with = "3.0"
+futures = "0.3"
+tsync = "2"
+chrono = { version = "0.4", features = ["serde"] }
+hyper = "0.14"
+
+# OpenAPI + Swagger
```

Comment:
Usually `swagger` is not included into release builds, thus I suggest to make it optional: add separate feature `swagger` that is enabled by default. In release build we can disable it